vitest.nuxt.config.ts

```diff
@@ -1,6 +1,11 @@
 import { defineVitestConfig } from '@nuxt/test-utils/config'
 
 export default defineVitestConfig({
+  resolve: {
+    alias: {
+      '#vue-router': 'vue-router'
```

Comment:
This shouldn't be needed, I think...

Reply:
check last failing commit: https://github.com/nuxt/nuxt/pull/26522/commits/302d91b3c78f548d0587e2e05b7b34c11210d0f0

Reply:
The reason this is failing is that we also have a version of Nuxt where there _is no_ vue-router. This 'universal' router needs to work when there is no `pages/` directory, so we can't directly import from or depend on vue-router in that case. We need to have a 'stub' version of `useLink` in that case...

Reply:
implementing - will push shortly

Reply:
the thing I pushed is an example implementation - but as you refactor to share logic across NuxtLink and useLink, you might find it needs to be updated.